docs/docs.scss

```diff
@@ -281,8 +281,8 @@ body {
   .column {
     padding: $grid-gutter;
     font-family: $mono-font;
-    background-color: lighten($brand-green, 40%);
-    border: 1px solid lighten($brand-green, 30%);
+    background-color: lighten($green, 40%);
+    border: 1px solid lighten($green, 30%);
```

Comment:
I agree, but I didn't add the lighten function here. Fixing all of these might be out of scope for this pull. I would gladly clean it up in a future pr though :+1: